packages/fccanalyses/package.py

```diff
@@ -111,11 +111,13 @@ def setup_build_environment(self, env):
     def setup_run_environment(self, env):
         env.prepend_path("ROOT_INCLUDE_PATH", self.prefix.include.FCCAnalyses)
         env.prepend_path("PYTHONPATH", self.prefix.python)
+        env.prepend_path("PYTHONPATH", self.prefix.share + '/examples')
         # this should point to share/ by key4hep convention
         #  but we want to make it work with the tutorials
         env.set("FCCANALYSES", self.prefix.python)
         env.prepend_path("LD_LIBRARY_PATH", self.spec["fccanalyses"].prefix.lib)
         env.prepend_path("LD_LIBRARY_PATH", self.spec["fccanalyses"].prefix.lib64)
+        env.prepend_path("FCCDICTSDIR", "/cvmfs/fcc.cern.ch/FCCDicts")
```

Comment:
```suggestion
        env.set("FCCDICTSDIR", "/cvmfs/fcc.cern.ch/FCCDicts")
```
I assume there are no more directories with dictionaries so setting the variable should be fine? It isn't very different but if another package were to prepend_path there could be a path more than once and maybe that breaks something